ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml

```diff
@@ -29,6 +29,17 @@ images:
       as:
       - e2e-tests
   to: e2e-tests
+- dockerfile_path: Dockerfile.logbridge
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/openshift-azure/logbridge
+    root:
```

Comment:
I think we don't need this
```yaml
root:
  as:
  - logbridge
```

Reply:
i think this is image build destination. we kinda have it for all other images.

Reply:
@bbguimaraes @droslean can you please verify whether we need `root`? Pretty sure @droslean was saying to me we didn't need it a while back.

Reply:
ci-operator is not expecting that field. For more information check [here](https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md)
So, you can remove it. The tag for the image is defined in `to` field.